modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java

```diff
@@ -235,6 +236,18 @@ public ObjectEntry getObjectEntry(Long objectEntryId) throws Exception {
 			objectEntryId);
 	}
 
+	@Override
+	public Page<Permission> getObjectEntryPermissionsPage(
+			Long objectEntryId, String roleNames)
+		throws Exception {
+
+		if (_objectDefinition.isRootDescendantNode()) {
```

Comment:
This would not be necessary if we include the operationId in the excluded operation ids

Reply:
Same here